src/backend/bs_srcserver

```diff
@@ -342,9 +342,9 @@ sub triggerservicerun {
   my ($cgi, $projid, $packid) = @_;
   my $rev = getrev($projid, $packid);
   my $pack = BSRevision::readpack_local($projid, $packid, 1);
-  if ($pack && $pack->{'title'} =~ /^obs-scm:(.*)/) {
+  if ($pack && $pack->{'scm'}) {
```

Comment:
```suggestion
  if ($pack && $pack->{'scmsync'}) {
```
Shouldn't it be `scmsync`? And also in the rest of the occurrences in the same file (I count five occurrences more).

Reply:
right, missed to change the second commit as well .. will take care